### PR TITLE
Make minimum staking amount as parameter

### PR DESCRIPTION
**Pull request by Jianzhuo**
close issue Fix the default minumStakingAmount #84

### CODE REVIEW

**Comment by mzxyz** [comment]
Only merge this PR after #86 merged and upgraded

**Review by mzxyz — APPROVED** [review]

File: scripts/config/local.config.ts [hunk 1]
```diff
@@ -47,6 +47,7 @@ export default function localConfig(): DeploymentConfig {
             EraManager: [60 * 60 * 24],
             ServiceAgreementRegistry: [1e6], //threshold
             PurchaseOfferMarket: [1e5, '0x0000000000000000000000000000000000000000'],
+            IndexerRegistry: [1000],
```
Comment (mzxyz) [inline scripts/config/local.config.ts]:
```suggestion
            IndexerRegistry: [1000e18],
```

File: test/setup.ts [hunk 2]
```diff
@@ -48,6 +48,7 @@ export const deployContracts = async (wallet: Wallet, wallet1: Wallet) => {
             EraManager: [60 * 60 * 24],
             PurchaseOfferMarket: [1e5, ZERO_ADDRESS],
             ServiceAgreementRegistry: [0], //threshold
+            IndexerRegistry: [1000],
```
Comment (mzxyz) [inline test/setup.ts]:
```suggestion
            IndexerRegistry: [1000e18],
```

**Review by mzxyz — APPROVED** [review]

**Comment by mzxyz** [comment]
## TODO: 

- [ ] Fix the test cases

File: scripts/config/testnet.config.ts [hunk 3]
```diff
@@ -14,5 +14,6 @@ export default {
         EraManager: [60 * 60], // 1 day
         ServiceAgreementRegistry: [1e6], //threshold
         PurchaseOfferMarket: [1e5, '0x0000000000000000000000000000000000000000'],
+        IndexerRegistry: ['1000000000000000000000'],
```
Comment (mzxyz) [inline scripts/config/testnet.config.ts:17]: Why not use `1000e18`, I think it would be more readable

**Review by mzxyz — APPROVED** [review]
Just one question, LGTM

File: scripts/config/testnet.config.ts (hunk 3 above)
Comment (Jianzhuo) [reply scripts/config/testnet.config.ts:17]: 1e18 will cause overflow to convert to BigNumber

**Pull request merged**